docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
 
 At this point, you should have a [fully-configured dbt project](/integrations/dbt/using-dbt-with-dagster/set-up-dbt-project) that's ready to work with Dagster.
 
-In this step, you'll finally begin integrating dbt with Dagster! To do this, you'll:
+In this section, you'll finally begin integrating dbt with Dagster! To do this, you'll:
 
-- [Load the dbt models into Dagster as assets](#step-1-load-the-dbt-models-as-assets)
-- [Define a Dagster code location](#step-2-define-a-dagster-code-location)
-- [View the assets in the Dagster UI](#step-3-view-the-assets-in-the-dagster-ui)
+- [Install Dagster and the Dagster-dbt library](#step-0-install-dagster-and-the-dagster-dbt-library)
+- [Create a Dagster project that wraps your dbt project](#step-1-create-a-dagster-project-that-wraps-your-dbt-project)
+- [Inspect your Dagster project in Dagster's UI](#step-2-inspect-your-dagster-project-in-dagsters-ui)
+- [Build your dbt project, aka materialize your assets](#step-3-build-your-dbt-project-aka-materialize-your-assets)
+- [Review the Python code in your Dagster project](#step-4-review-the-python-code-in-your-dagster-project)
 
 ---
 
-## Step 1: Load the dbt models as assets
+## Step 0: Install Dagster and the Dagster-dbt library
 
-In this step, you'll load the dbt models into Dagster as assets using the [`dagster-dbt`](/\_apidocs/libraries/dagster-dbt) library.
+```shell
+pip install dagster dagster-webserver dagster-dbt
+```
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster/assets`, and add the following code:
+---
 
-```python
-from dagster_dbt import load_assets_from_dbt_project
+## Step 1: Create a Dagster project that wraps your dbt project
 
-from dagster import file_relative_path
+You can create a Dagster project that wraps your dbt project by using the `dagster-dbt` command line interface. Make sure you're in the directory where your dbt_project.yaml is. If you're continuing from the previous section, then you'll already be in this directory. Then, run:
 
+```shell
+dagster-dbt project scaffold --project-name jaffle_dagster
+```
 
-DBT_PROJECT_PATH = file_relative_path(__file__, "../../jaffle_shop")
+This creates a directory called `jaffle_dagster/` inside the current directory. The `jaffle_dagster/` directory contains a set of files that define a Dagster project.
```

Comment:
Should `a set of files that define a Dagster project` link to the project files guide?: `/getting-started/project-file-reference`

Reply:
Oh this is a good idea but annoyingly the project files in our dbt scaffold are a little different than the files in our default project. I added a reminder to add this link when we make them the same: https://github.com/dagster-io/dagster/issues/15112#issuecomment-1649068100.